client/my-sites/stats/stats-subscribers-page/subscribers-highlight-section.jsx

```diff
@@ -0,0 +1,93 @@
+import { CountComparisonCard, Popover } from '@automattic/components';
+import { Icon, info } from '@wordpress/icons';
+import { translate } from 'i18n-calypso';
+import { useRef, useState } from 'react';
+import './subscribers-highlight-section.scss';
+
+function getSubscriberHighlights() {
```

Comment:
We could convert this into a hook function by renaming it `useSubscriberHighlights` and optionally using a [useMemo hook](https://react.dev/reference/react/useMemo) to memorize its contents.

Reply:
I'll push a commit to address this.